src/common/options.cc

```diff
@@ -3134,7 +3134,7 @@ std::vector<Option> get_global_options() {
     .add_see_also("osd_pg_log_dups_tracked"),
 
     Option("osd_max_pg_log_entries", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
-    .set_default(3000)
+    .set_default(10000)
```

Comment:
I thought an earlier commit (PR 20394)  had lowered this from 10000 to 3000.   Isn't the spread between min and max pg log entries the amount that has to be buffered by the OSD process?      But perhaps I'm getting branches confused or something like that?  For background on this, see [this article](https://docs.google.com/document/d/1e2jn8DbVbpwYcuhPG18tP3tDzaJ6DBHiacdzqiZcFow/edit#heading=h.x4uti0xeq736) (available to non-Red Hat people on request).